examples/demo/landing_page_demo_2.tut

```diff
@@ -4,9 +4,11 @@ source config.sh.in
 
 run() {
     c PyFunceble --version
-    c PyFunceble -d 192.168.178.60 # TODO doesn't print the status
-    c PyFunceble -d 192.168.178.250 # TODO doesn't print the status
-    c PyFunceble -d 192.168.178.259
+    c PyFunceble -d 9.9.9.9                         # Neutral stable
+    c PyFunceble -d 2620:fe::fe                     # quad9 IPv6 (As I recall GHA now supports IPv6)
+    c PyFunceble -d 194.187.99.221                  # Adult CDN: https://mypdns.org/my-privacy-dns/porn-records/-/issues?scope=all&state=all&label_name[]=IP%3A%3ABlackListing
+    c PyFunceble -d 45.136.204.40                   # 23 phis: https://github.com/mitchellkrogza/phishing/pull/67
```

Comment:
Why not merge these two lines to demonstrate how multiply values are committed at once?

`c PyFunceble -d 194.187.99.221 45.136.204.40` :thought_balloon: :thinking: , could that be an idea?? @funilrys and @veracioux